assignments/array-methods.js

```diff
@@ -56,28 +56,63 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function(runner) {
+    fullName.push(runner.first_name + " " + runner.last_name);
```

Comment:
well done, I would suggest that once you get through the problems, try going back and seeing how you could apply es6 syntax to it to get used to it. This could be written out as follows;

`let fullName = runners.forEach((runner) => `${runner.first_name} ${runner.last_name}`

^^ this makes it much easier to read and a lot less writing in the end.